app/components/channel_list/categories/categories.tsx

```diff
@@ -44,6 +48,13 @@ const Categories = (props: Props) => {
             style={styles.flex}
             showsHorizontalScrollIndicator={false}
             showsVerticalScrollIndicator={false}
+            keyExtractor={extractKey}
```

Comment:
In terms of time or memory usage, what is the gain here ?

Reply:
what is your question about @avinashlng1080 ?

> Used to extract a unique key for a given item at the specified index. Key is used for caching and as the react key to track item re-ordering. The default extractor checks item.key, then item.id, and then falls back to using the index, like React does.

Reply:
I was referring to the whole optimizations not this 'extractKey'.  Just wanted to know by how much we are winning with this new implementation.